modules/installation-mirror-repository.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_disconnected/installing-disconnected-preparations.adoc
+
+[id="installation-mirror-repository_{context}"]
+= Mirroring the {product-title} image repository
+
+Mirror the {product-title} image repository to use during cluster installation
+or upgrade.
+
+.Prerequisites
+
+* You prepared the bastion host to use in a disconnected installation.
+* You configured a mirror registry to use in your disconnected environment and
+can access the certificate and credentials that you configured.
+* You downloaded the pull secret from the
+link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers]
+page and modified it to include authentication to your mirror repository.
+
+.Procedure
+
+Complete the following steps on the bastion host:
+
+. Review the
+link:https://quay.io/repository/openshift-release-dev/ocp-release?tab=tags[Quay.io]
+repository to determine the version of {product-title} that you want to install.
+
+. Set the required environment variables:
++
+----
+$ export OCP_RELEASE=<release_version> <1>
+$ export AIRGAP_REG='<bastion_host_name>:5000' <2>
+$ export AIRGAP_REPO='<repo_name>' <3>
+$ export PRODUCT_REPO='openshift-release-dev' <4>
+$ export AIRGAP_SECRET_JSON='<path_to_pull_secret>${HOME}/pull-secret-2.json' <5>
```

Comment:
nit: this seems to mix `<placeholder>` with an example value.  For the other variables, you seem to just use `<placeholder>` and then give example values down below under `<1>` and similar.

Reply:
That's because the instructions specified a specific value for the pull secret file - you pointed out an improvement to it in a previous comment. I'm adjusting all of the pull secret file names to be generic.